subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/execution/PluginsBlockInterpreterTest.kt

```diff
@@ -413,15 +413,15 @@ class PluginsBlockInterpreterTest {
     fun `id version - comment - kdoc inline`() {
         assertStaticInterpretationOf(
             """
-                id("plugin-id-6") /** kdoc comment */ .version("1.0")
-                id("plugin-id-7"). /** kdoc comment */ version("1.0")
-                id("plugin-id-8") /** kdoc comment */ version "1.0"
-                id("plugin-id-9") version /** kdoc comment */ "1.0"
-            """,
-            PluginRequestSpec("plugin-id-6", version = "1.0"),
-            PluginRequestSpec("plugin-id-7", version = "1.0"),
-            PluginRequestSpec("plugin-id-8", version = "1.0"),
-            PluginRequestSpec("plugin-id-9", version = "1.0"),
+                id("plugin-id-1") /** kdoc comment */ .version("1.0")
+                id("plugin-id-2"). /** kdoc comment */ version("2.0")
+                id("plugin-id-3") /** kdoc comment */ version "3.0"
+                id("plugin-id-4") version /** kdoc comment */ "4.0"
+            """.trimIndent(),
```

Comment:
❔ Why adding `.trimIndent()`?

Reply:
I wanted to say "why not", but now I see that it's not used consistently. Let me fix that.